NEWS.md

```diff
@@ -137,6 +219,8 @@ contributors as noted in the log.
 
 ### Changed
 
+- In theme settings, mark color definitions in the format #rrggbb aaa` as
+  deprecated (still supported, but will removed in some future release).
```

Comment:
Suggest adding something like "in favor of the more commonly used `#rrggbbaa` format"